gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/pod/tasks/CreatePodYAMLTask.groovy

```diff
@@ -56,14 +57,14 @@ class CreatePodYAMLTask extends DefaultTask {
     void createPodYAML() {
         validatePodYAML()
         getPodYAMLFile().setText(JsonOutput.prettyPrint(JsonOutput.toJson([
-                'services': jsonMapper.convertValue(this.serviceDefinitions, Map),
-                'volumes': jsonMapper.convertValue(this.volumeDefinitions, Map),
+                'services': jsonMapper.convertValue(this.serviceDefinitions, new TypeReference<Map<Object, Object>>() {}),
```

Comment:
can you make this a `Map<String, Object>` (and in the next line) since json keys must be strings?

Reply:
Sure